### ISSUE TITLE
use loss = 'MultiClass' for catboost multiclass

### ISSUE THREAD

**Issue by ClimbsRocks** [ClimbsRocks/auto_ml#302]
https://tech.yandex.com/catboost/doc/dg/concepts/python-reference_catboostclassifier-docpage/

cls = catboost.CatBoostClassifier(loss_function='MultiClass')
